R/scatter_histogram.R

```diff
@@ -0,0 +1,51 @@
+#' scatter_histogram
+#'
+#' Create histogram from given point coordinates.
+#'
+#' @param xy 2-column float matrix with point coordinates. As usual with
+#'           rasters in R, X axis grows right, and Y axis grows DOWN.
+#'           Flipping `ylim` causes the "usual" mathematical behavior.
+#'
+#' @param xlim Float limits as usual (position of the first pixel on the
```

Comment:
"types" in R aren't much necessary :D

Reply:
So you mean that I should delete all types such as floats, integers etc. from parameter descriptions?

Reply:
yeah I think R doesn't even typeset these as types, see https://teri934.github.io/scattermore2/reference/scatter_histogram.html

Reply:
Ok, thanks